planning/motion_velocity_smoother/src/trajectory_utils.cpp

```diff
@@ -212,22 +212,31 @@ std::vector<double> calcTrajectoryCurvatureFrom3Points(
 
   // calculate curvature by circle fitting from three points
   std::vector<double> k_arr;
-  for (size_t i = idx_dist; i < trajectory.size() - idx_dist; ++i) {
+  //  for first point curvature = 0;
+  k_arr.push_back(0.0);
```

Comment:
@brkay54 For the first and last points, how about taking a copy from the next/previous value?

```
std::vector<double> k_arr(trajectory.size(), 0.0);

...
// compute k for each index
...

k_arr.at(0) = k_arr.at(1);
k_arr.back() = k_arr.at(trajectory.size() - 2);
```